public_html/php/oauth.php

```diff
@@ -186,9 +186,11 @@ function fetchAccessToken() {
 		$_SESSION['tokenKey'] = $this->gTokenKey = $token->key;
 		$_SESSION['tokenSecret'] = $this->gTokenSecret = $token->secret;
 		if ( $this->use_cookies ) {
-			$t = time()+60*60*24*30*3 ; // expires in three months
+			$t = time();
```

Comment:
`session_set_cookie_params` expects a duration as the expiry and `setcookie` expects a timestamp.